config/initializers/100-i18n.rb

```diff
@@ -4,7 +4,3 @@
 I18n.backend = I18n::Backend::DiscourseI18n.new
 I18n.config.missing_interpolation_argument_handler = proc { throw(:exception) }
 I18n.init_accelerator!
```

Comment:
I'm not sure if this is intentional, but `I18n.reload!` is called twice during initialization. Once by this call to `I18n.init_accelerator!` and once by [i18n_railtie.rb](https://github.com/rails/rails/blob/7d242be5a4fbfaea2d585905d76cc6bae45cbe77/activesupport/lib/active_support/i18n_railtie.rb#L63). I thought I'd mention it since I noticed it during debugging.